source/_integrations/home_connect.markdown

```diff
@@ -111,6 +85,741 @@ This integration follows standard integration removal. No extra steps are requir
 
 After deleting the integration, go to [your applications at the Home Connect Developer portal](https://developer.home-connect.com/applications), find the application that you were using for Home Assistant, click on details and click on "Delete Application".
 
+## Supported functionality
+
+{% note %}
+
+- The entities availability depends on the appliance type, but the appliance might not support all the entities for its type.
+- Some appliances don't report data while they are turned off so corresponding entities will not appear in the Home Connect integration after loading until the appliances are turned on.
+{% endnote %}
+
+### Binary sensor
+
+- **Connectivity**:
+  - **Description**: Shows the connectivity status of the appliance.
+  - **Availability**: All appliances
+- **Remote control**:
+  - **Description**: Indicates if the remote control is enabled.
+  - **Availability**: Cooktop, Hood, Oven, Warming Drawer, Dishwasher, Washer, Dryer, Washer Dryer
+- **Remote start**:
+  - **Description**: Indicates if a program can be started remotely.
+  - **Availability**: Coffee maker, Hood, Oven, Warming Drawer, Dishwasher, Washer, Dryer, Washer Dryer
+- **Local control**:
+  - **Description**: Indicates whether the home appliance is currently physically controlled by the user.
+  - **Availability**: Coffee maker, Cooktop, Hood, Oven, Warming Drawer, Washer, Dryer, Washer Dryer
+- **Bottle cooler door**:
+  - **Description**: Indicates if the bottle cooler door is open.
+  - **Availability**: Fridge freezer, Refrigerator
+- **Chiller door**:
+  - **Description**: Indicates if the chiller door is open.
+  - **Availability**: Fridge freezer, Refrigerator
+- **Flex compartment door**:
+  - **Description**: Indicates if the flex compartment door is open.
+  - **Availability**: Fridge freezer
+- **Freezer door**:
+  - **Description**: Indicates if the freezer door is open.
+  - **Availability**: Freezer, Fridge freezer
+- **Refrigerator door**:
+  - **Description**: Indicates if the refrigerator door is open.
+  - **Availability**: Fridge freezer, Refrigerator
+- **Wine compartment door**:
+  - **Description**: Indicates if the wine compartment door is open.
+  - **Availability**: Wine cooler
+- **Battery charging state**:
+  - **Description**: Describes if the appliance is charging or discharging.
+  - **Availability**: Cleaning robot
+- **Charging connection**:
+  - **Description**: Indicates if the appliance is connected or disconnected.
+  - **Availability**: Cleaning robot
+
```

Comment:
```suggestion

{% enddetails" %}

```
As these lists are rather long, maybe we could wrap them in a collapsible. What do you think?

Reply:
You mean all the lists?
I looks cleaner without details section, but adding details to all the list makes easier to navigate through the documentation (specially for mobile devices, as the "On this page" section is on the bottom and cannot be accessed from anywhere in the page)

Reply:
yes, I meant all of them. only added the suggestion to one as an example.

Reply:
Thank you for all the tweaks! I still think all these long lists should go into a details block